internal/graph/check.go

```diff
@@ -36,13 +36,29 @@ type ResolveCheckRequest struct {
 	TupleKey             *openfgav1.TupleKey
 	ContextualTuples     []*openfgav1.TupleKey
 	Context              *structpb.Struct
-	ResolutionMetadata   *ResolutionMetadata
+	RequestMetadata      *CheckRequestMetadata
```

Comment:
```suggestion
	Metadata      *CheckRequestMetadata
```

RequestMetadata stutters - we can simplify to just Metadata since it is attached to the request to begin with.

Reply:
I don't want to do this change now because it will blow up this PR with too many lines changed

Reply:
Ok, let's follow up semi-promptly at least.

Reply:
Sounds good!